### PR TITLE
RFC: Remove Cluster and WSFS views

### PR DESCRIPTION
**Pull request by fjakobs**
This PR removes the cluster and WSFS views as they are not project centric and don't really add to the inner dev loop experience.

Similar functionality can be provided by 3rd party extensions like https://marketplace.visualstudio.com/items?itemName=paiqo.databricks-vscode

### CODE REVIEW

**Comment by fjakobs** [comment]
@kartikgupta-db I wonder if we can also simplify the cluster manager after merging this PR

File: packages/databricks-vscode/src/cluster/ClusterTreeNode.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        label: "Driver",
+        description: element.details.driver_node_type_id,
+    });
+
+    if (element.details.autoscale) {
+        children.push({
+            label: "Worker",
+            description: `${element.details.node_type_id}, ${element.details.autoscale.min_workers}-${element.details.autoscale.max_workers} workers`,
+        });
+    } else if (element.details.num_workers || 0 > 0) {
```
Comment (kartikgupta-db) [inline packages/databricks-vscode/src/cluster/ClusterTreeNode.ts:70]: 0 > 0 ?

File: packages/databricks-vscode/src/extension.ts [hunk 2]
```diff
@@ -106,30 +105,17 @@ export async function activate(
...
-        window.registerTreeDataProvider(
-            "workspaceFsView",
-            workspaceFsDataProvider
-        ),
         commands.registerCommand(
             "databricks.wsfs.attachSyncDestination",
             workspaceFsCommands.attachSyncDestination,
             workspaceFsCommands
         ),
-        commands.registerCommand(
```
Comment (kartikgupta-db) [inline packages/databricks-vscode/src/extension.ts:128]: We can remove wsfs.attachSyncDestination as well and remove these commands from package.json.

**Review by kartikgupta-db — COMMENTED** [review]
This does simplify a bunch of things!

File: packages/databricks-vscode/src/extension.ts (hunk 2 above)
Comment (fjakobs) [reply packages/databricks-vscode/src/extension.ts:128]: +1 will do

**Pull request closed without merging**